common/meson.build

```diff
@@ -0,0 +1,89 @@
+install_headers('pkcs11.h', 'pkcs11x.h', subdir: 'p11-kit-1/p11-kit')
+
+libp11_common_sources = [
+  'argv.c', 'argv.h',
```

Comment:
It is not necessary to list headers in source file list.

Reply:
Removed.